.github/workflows/tests.yml

```
@@ -346,12 +353,19 @@ jobs:
         # NB: mesa-utils is needed for PyQT testing (https://github.com/Microsoft/WSL/issues/1246#issuecomment-356425862)
         sudo apt update && sudo DEBIAN_FRONTEND=noninteractive apt install -y gcc git curl mesa-utils
     - uses: actions/checkout@v2
+    - name: Copy SCT from $GITHUB_WORKSPACE to ~
+      run: |
+        cd ..
+        cp -r spinalcordtoolbox ~/spinalcordtoolbox
```

If we don't do

```suggestion
        cp -rp spinalcordtoolbox ~/spinalcordtoolbox
```

there will be even more mysterious permission errors down the road.

---

Thank you for the input re: bash wizardry